website/content/en/preview/concepts/nodeoverlays.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "NodeOverlays"
+linkTitle: "NodeOverlays"
+weight: 40
+description: >
+  Understand NodeOverlays and how they enable fine-tuning of Karpenter's scheduling simulation for advanced use cases.
+---
+
+Karpenter uses NodeOverlays to inject alternative assumptions into the scheduling simulation for more accurate scheduling decisions.
+NodeOverlays enable users to fine-tune instance pricing and add extended resources to instance types during Karpenter's decision-making process.
+They provide a flexible way to account for real-world factors like savings plans, licensing costs, and custom hardware resources that aren't captured in the base instance data from cloud providers.
+
+NodeOverlays work by modifying the instance type information that Karpenter uses during its scheduling simulation.
+When Karpenter evaluates which instance types can satisfy pending pod requirements, it applies any matching NodeOverlays to adjust pricing information or add extended resources before making provisioning decisions.
+
+## NodeOverlay Configuration
+
+```yaml
+apiVersion: karpenter.sh/v1alpha1
+kind: NodeOverlay
+metadata:
+  name: example-overlay
+spec:
+  # Optional weight for conflict resolution (higher weight wins)
+  weight: 10
+  
+  # Requirements determine which instance types this overlay applies to
+  requirements:
```

Comment:
Can you add an example of a non-instance type requirement that this would apply to? That's what the doc implies on line 50.

Reply:
Added a few examples that were not based on the